FoodStandardsAgency/food-gov-uk
# Develop: Upgrade Safeword module to version 7.x-1.13

Upgraded the Safeword module to version **7.x-1.13**

This module provides a field type for generating URL-safe strings for use by Pathauto in creating aliases for nodes.

Database schema updates
-----------------------
No database schema updates are required.

Testing
-------
This module is used primarily to provide the Path field that is present in most node types on the FSA site. These are used to generate the final part of the URL for the page.

Local testing has not revealed any problems.

Suggested testing:

1. Create nodes of a few different types and verify that their URLs are based on a sanitised version of the title field content
2. Edit the Path field for existing nodes and verify that the URL for the page changes accordingly.
3. Also, following 2, verify that URL redirects are generated for the previous node URLs
4. For an existing node, click the edit button for the path field and clear its contents. Save the page and verify that the URL changes.

[ Partial fix for [#10115](https://support.siriusopensource.com/index.pl?Action=AgentTicketZoom;TicketID=613) ]
